about.html

```diff
@@ -15,28 +15,28 @@
   <div class="wrapper">
     <header>
 
```

Comment:
- [ ] As requested by the previous reviewers, From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), 
  - On the home page, the text `The long-awaited fashion event is finally here.Always an exciting ...` have a background color and design in Cindy's design you that don't have. :blush: 
  - You are not using the same colors in the Cindy's design as requested by the previous reviewer. you are using  `blue`, `purple`, `gold`, and `violet` instead of using ![image](https://user-images.githubusercontent.com/92303891/226748308-b1dbc0a9-ec12-472e-bde9-1ee313dd5a3e.png)


| Your Design |Cindy Design |
| :---:   | :---: | 
| ![image](https://user-images.githubusercontent.com/92303891/226746829-668fb2ca-6d4a-4e5d-a63b-9f8ce0e6480a.png) | ![image](https://user-images.githubusercontent.com/92303891/226747165-fd08041f-7544-4367-9744-12209089b205.png) |


- [ OPTIONAL ] Also on the about page the text `A fashion week is a fashion industry event, lasting approximately one week, where ..` have a background color that you don't have in your design so make sure that you adjust the design to look precisely as Cindy's design. :relaxed: 


| Your Design |Cindy Design |
| :---:   | :---: | 
| ![image](https://user-images.githubusercontent.com/92303891/226745506-4092a779-1908-40d7-a3ba-9b50591f41b1.png) |  ![image](https://user-images.githubusercontent.com/92303891/226746099-c6365ba7-d494-4e35-a35d-9217fb850697.png)  |